core/test/utils/api.js

```diff
@@ -61,6 +61,22 @@ function getApiQuery(route) {
     return url.resolve(ApiRouteBase, route);
 }
 
+function getApiPath(options) {
+    const baseAPIPath = '/ghost/api/';
+    switch(options.version) {
+        case 'deprecated':
+            return `${baseAPIPath}v0.1/`;
+        case 'active':
+            if (options.versionType === 'admin') {
+                return `${baseAPIPath}v2/admin/`;
+            } else {
+                return `${baseAPIPath}v2/content/`;
+            }
+        default:
+            return `${baseAPIPath}v0.1/`;
+    }
+}
+
```

Comment:
Created a separate method to be used in tests as this seemed more standard approach to compare test case values with different version rather than using the same method in result as that doesn't seem to be much useful.